extensions/azurecore/package.nls.json

```
@@ -20,14 +20,33 @@
 	"config.azureAccountConfigurationSection": "Azure Account Configuration",
 	"config.enablePublicCloudDescription": "Should Azure public cloud integration be enabled",
 	"config.enableUsGovCloudDescription": "Should US Government Azure cloud (Fairfax) integration be enabled",
-	"config.enableUsNatCloudDescription": "Should US National Azure cloud integration be enabled",
 	"config.enableChinaCloudDescription": "Should Azure China integration be enabled",
-	"config.enableGermanyCloudDescription": "Should Azure Germany integration be enabled",
 	"config.azureAuthMethodConfigurationSection": "Azure Authentication Method",
 	"config.azureCodeGrantMethod": "Code Grant Method",
 	"config.azureDeviceCodeMethod": "Device Code Method",
 	"config.noSystemKeychain": "Disable system keychain integration. Credentials will be stored in a flat file in the user's home directory.",
 	"config.piiLogging": "Should Personally Identifiable Information (PII) be logged in the Azure Accounts output channel and the output channel log file.",
 	"config.loggingLevel": "[Optional] The verbosity of logging for the Azure Accounts extension.",
-	"config.authenticationLibrary": "The library used for the AAD auth flow. Please restart ADS after changing this option."
+	"config.authenticationLibrary": "The library used for the AAD auth flow. Please restart ADS after changing this option.",
+	"config.providerSettingsJson": "Setting containing custom Azure authentication endpoints. Changes to this setting require a restart to take effect.",
+	"config.providerSettingsTitle": "Provider Settings",
+	"config.providerSettingsName": "Cloud Name",
+	"config.providerSettingsDescription": "Cloud Settings",
+	"config.providerSettingsMetadata": "Cloud Display Name",
+	"config.providerSettingsId": "Cloud ID",
+	"config.providerSettings.endpoints.host": "Host Endpoint",
+	"config.providerSettings.endpoints.clientId": "Client ID for Azure Data Studio",
+	"config.providerSettings.endpoints.microsoftResource": "Microsoft Resource Endpoint",
+	"config.providerSettings.endpoints.graphResource": "Graph Resource Endpoint",
+	"config.providerSettings.endpoints.msGraphResource": "Microsoft Graph Resource Endpoint",
+	"config.providerSettings.endpoints.armResource": "ARM Resource Endpoint",
+	"config.providerSettings.endpoints.sqlResource": "SQL Resource Endpoint",
+	"config.providerSettings.endpoints.azureKeyVaultResource": "Azure KeyVault Resource Endpoint",
+	"config.providerSettings.endpoints.logAnalytics": "Azure Log Analytics Endpoint",
+	"config.providerSettings.endpoints.azureStorageResource": "Azure Storage Resource Endpoint",
+	"config.providerSettings.endpoints.azureStorageResourceSuffix": "Azure Stroage Resource Endpoint Suffix",
```

(if you don't have a spell check extension installed in VS Code, I'd suggest adding that to help avoid stuff like this)

```suggestion
	"config.providerSettings.endpoints.azureStorageResourceSuffix": "Azure Storage Resource Endpoint Suffix",
```